ci/do_ci.sh

```diff
@@ -34,6 +34,36 @@ function test_salvo() {
   popd
 }
 
+# Check the Salvo python files format
+function check_format() {
+  echo "Checking the Salvo python files format"
+  pushd salvo
+
+  tools/format_python_tools.sh check
```

Comment:
Did we verify that this fails when some files are formatted incorrectly?

Reply:
Yes, I think so, it will exit with 1 if it fails: https://github.com/envoyproxy/envoy-perf/blob/e1804e7cd6a94e4437095b5f2e5d560034989b08/salvo/tools/format_python_tools.py#L75, just like other ci task do.